File: easy/jungle/jungle.java
package ejercicios;

import java.util.Scanner;

/**
 *06/05/2021
 * @author Andre Peralta
 */
public class metodo {
    public static void main(String[] args) {
        /*Esto es para que acepte una entrada con espacios: 
        ejemplo: Rawr Chirp Ssss*/        
        Scanner scan = new Scanner(System.in);
        String noises = "";
        noises += scan.nextLine();
        scan.close();

        //Separamos el string por espacios
        String[] NoisesList = noises.split("\\s+");
 
        String ouput = "";
        
        
        for(String item : NoisesList){

            switch (item) 
            {
                case "Grr":  ouput = ouput + "Lion ";
                         break;
                case "Rawr":  ouput = ouput + "Tiger ";
                         break;
                case "Ssss":  ouput = ouput + "Snake ";
                         break;
                case "Chirp":  ouput = ouput + "Bird ";
                         break;
            }
        }
        
        System.out.println(ouput);
         
    }
    
}

File: easy/jungle/jungle.rb
"""
@author	André Peralta
@version 20/05/2021
"""

def noises_hear noises

	noises = noises.split(' ')

	for i in noises

		case i
		when 'Grr'
			print "Lion "

		when 'Rawr'
			print "Tiger "

		when 'Ssss'
			print "Snake "

		when 'Chirp'
			print "Bird "

		end
	end
end


puts "Noises: "
noises = gets

noises_hear(noises)

